ddtrace/debugging/_uploader.py

```diff
@@ -76,7 +75,6 @@ def _write(self, payload):
                 else:
                     meter.increment("upload.success")
                     meter.distribution("upload.size", len(payload))
-                    log.debug("[%s][P: %s] Payload uploaded: %s", os.getpid(), os.getppid(), payload)
```

Comment:
Same as above

Reply:
I added those logs in a [related PR](https://github.com/DataDog/dd-trace-py/pull/5464) but I noticed those logs are redundant and make too much noise :+1: